src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -9827,6 +9845,14 @@ private StatementSyntax ParseLocalDeclarationStatement(SyntaxList<AttributeListS
             var variables = _pool.AllocateSeparated<VariableDeclaratorSyntax>();
             try
             {
+                SyntaxToken scopedKeyword = ParsePossibleScopedKeyword(isFunctionPointerParameter: false);
+
+                // For local functions, 'scoped' is a modifier in LocalFunctionStatementSyntax
```

Comment:
Is an error reported if a local function has the scoped modifier?

Reply:
> Is an error reported if a local function has the scoped modifier?

Yes. There is a test for that. ```ReturnTypeScope``` in RefFieldTest.cs, for example.